channels/pe.m3u

```diff
@@ -61,6 +61,8 @@ https://cp.sradiotv.com:1936/Stream/Stream/playlist.m3u8
 https://stmv.panel.mivideo.pro/mileniuntv/mileniuntv/playlist.m3u8
 #EXTINF:-1 tvg-id="MonterricoTV.pe" tvg-name="Monterrico TV" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://yt3.ggpht.com/ytc/AAUvwnhD4JjitkowIeOGRlCsK-kmDk1T3lXgqklUjSv5KA=s320-c-k-c0x00ffffff-no-rj" group-title="Sports",Monterrico TV (1080p)
 http://vs8.live.opencaster.com/20100152275/jcpstream/playlist.m3u8
+#EXTINF:-1 tvg-id="MonterricoTV.pe" tvg-name="Monterrico TV" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://yt3.ggpht.com/ytc/AAUvwnhD4JjitkowIeOGRlCsK-kmDk1T3lXgqklUjSv5KA=s320-c-k-c0x00ffffff-no-rj" group-title="Sports",Monterrico TV (720p)
+https://www.opencaster.com/resources/hls_stream/hipodromojcp2.m3u8
```

Comment:
![image](https://user-images.githubusercontent.com/7253922/120190088-cb457180-c220-11eb-8595-25a9d2d81d7c.png)

Reply:
The website is https://hipodromodemonterrico.com.pe/monterrico/home.  I put Not 24/7 to this channel. With this I am evidencing that not all channels broadcast in the evenings or Sundays. Anyway, you can consult the website for the source of the m3u.